src/Explore/ScoreIndicator.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+
+import { Box } from '../Primitives'
+
+function ScoreIndicator({ score }) {
+  const [text, setText] = useState(score.toFixed(3))
+
+  const handleHoverStart = () => {
+    setText('Relevancy')
+  }
+
+  const handleHoverEnd = () => {
+    setText(score.toFixed(3))
+  }
+
+  return (
+    <Box
+      sx={{
+        position: 'relative',
+        width: 96,
+        background: 'linear-gradient(to right, #bb4677, #82b8e4)',
+        borderRadius: 8,
+        marginLeft: 1,
+        overflow: 'hidden',
+      }}
+      onMouseEnter={handleHoverStart}
+      onTouchStart={handleHoverStart}
+      onMouseLeave={handleHoverEnd}
+      onTouchEnd={handleHoverEnd}
+    >
+      <Box
+        sx={{
+          position: 'relative',
+          width: '100%',
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
```

Comment:
`alignItems: center` should work as well?

Reply:
Do you mean simply removing the `flexDirection`?

Reply:
Actually, you may need to keep `justifyContent: 'center'` and remove `textAlign: center`. Oh well, not a big deal, it's fine either way.

Reply:
Seems to be resulting to the following, if I understand you correctly.

![image](https://user-images.githubusercontent.com/24203905/173818870-719f8874-6d64-47be-8be4-c5c259484070.png)

Reply:
Yeah, sorry I should have tested and given you a clearer code change suggestion -- don't worry about it, `flexDirection` is fine anyway.